packages/verifiable-credentials/src/credentials.ts

```diff
@@ -206,21 +218,38 @@ export default class Credentials {
 			},
 		};
 
+
+
+
 		if (options && options.expirationDate) {
 			// The DID JWT VC library (called by createVerifiableCredential) verifies the string format so we do not need a test for that
+			const dateFormat = dayjs(options.expirationDate).utc(true)
+			if (dateFormat.$d.toString() === 'Invalid Date') {
+				throw new Error('Date format does not match ISO standard')
+			}
 			vcPayload.expirationDate = options.expirationDate
 		}
 
 		if (options && options.issuanceDate) {
-			vcPayload.issuanceDate = options.issuanceDate
+			const dateFormat = dayjs(options.issuanceDate).utc(true)
+			if (dateFormat.$d.toString() === 'Invalid Date') {
+				throw new Error('Date format does not match ISO standard')
+			}
+
+			vcPayload.issuanceDate = dateFormat.$d
 		} else {
-			vcPayload.issuanceDate = new Date().toISOString()
-		}
+			const currentTime = dayjs(new Date().toISOString()).utc(true).second();
+			const createdVcTime = dayjs(vcPayload.issuanceDate).utc(true).second()
+			const MAX_DURATION_IN_SECONDS = 5
 
+			if ((currentTime - createdVcTime) > MAX_DURATION_IN_SECONDS) {
```

Comment:
@cmcWebCode40 This is incorrect

You need a unit test that checks the `issuanceDate` is created within 5 seconds of the current time.

We don't want to restrict credentials to be created within a particular time frame.

Reply:
@tahpot, 

here is the test case i wrote for this https://github.com/verida/verida-js/blob/1e99a864f4d141afe928954157896921db65f623/packages/verifiable-credentials/test/create-credentials.test.ts#L110

is this test case incorrect ?

Reply:
Yes, that test case is correct (apart from my comment below)

Reply:
I think I'm a little bit confused on how exactly how to implement this feature